core/core-impl/pom.xml

```diff
@@ -30,23 +30,41 @@
     </dependency>
     <dependency>
       <groupId>io.fabric8.launcher</groupId>
-      <artifactId>launcher-service-git-impl</artifactId>
-      <type>test-jar</type>
-      <scope>test</scope>
+      <artifactId>launcher-service-openshift-impl</artifactId>
     </dependency>
     <dependency>
       <groupId>io.fabric8.launcher</groupId>
-      <artifactId>launcher-service-openshift-impl</artifactId>
+      <artifactId>launcher-service-keycloak-impl</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.jboss.forge.addon</groupId>
+      <artifactId>resources-api</artifactId>
```

Comment:
Replace with the Forge addon in this case (which will bring the impl) : 

```
    <dependency>
      <groupId>org.jboss.forge.addon</groupId>
      <artifactId>resources</artifactId>
      <classifier>forge-addon</classifier>
    </dependency>
```

Same for the maven and parser-xml dependencies below

Reply:
Is this ok?
```
<dependency>
      <groupId>org.jboss.forge.addon</groupId>
      <artifactId>resources</artifactId>
      <classifier>forge-addon</classifier>
    </dependency>
    <dependency>
      <groupId>org.jboss.forge.addon</groupId>
      <artifactId>maven</artifactId>
      <classifier>forge-addon</classifier>
    </dependency>
    <dependency>
      <groupId>org.jboss.forge.addon</groupId>
      <artifactId>parser-xml</artifactId>
      <classifier>forge-addon</classifier>
    </dependency>
```

Reply:
Yes, you can see that in the launcher-addon pom.xml too.